src/components/Excelerator/Excelerator.js

```diff
@@ -0,0 +1,91 @@
+import XLSX from "xlsx";
+import _ from "lodash";
+
+export function getCsvHeader(fields) {
+    const listNamesInGqlObject = getListNamesOfGqlObject(fields);
+
+    const listObjects = fields.filter(({ name }) => listNamesInGqlObject.includes(name));
+    const nonListObjects = fields.filter(({ name }) => !listNamesInGqlObject.includes(name));
+
+    // Create 3 columns for each list object
+    const objectCsvData = listObjects.map(({ name }) => {
+        return [1, 2, 3].map(i => ({ [`${name}${i}`]: {} }));
+    });
+
+    const nonObjectCsvData = nonListObjects.map(({ name }) => ({ [name]: {} }));
+
+    return [...nonObjectCsvData, ...objectCsvData.flat()];
+}
+
+export function createCsv(fileName, data) {
+    const newWorkbook = XLSX.utils.book_new();
+
+    const worksheet = XLSX.utils.json_to_sheet(data);
+    XLSX.utils.book_append_sheet(newWorkbook, worksheet);
+
+    XLSX.writeFile(newWorkbook, `${fileName}.csv`);
+}
```

Comment:
These methods are no longer in use but I still kept them here in case will need it (did not apply your comment)